client/noise/socketio/SocketIOManager.swift

```diff
@@ -73,42 +93,41 @@ class SocketIOManager: NSObject {
         socket.emit("signUp", user)
     }
     
+    // TODO: send encrypted message
+    func sendEncryptedChat(message: Dictionary<String, AnyObject>){
+        socket.emit("encryptedChatSent", message)
+    }
+    
     func retrieveMessages(userID: Int, friends: Dictionary<String, Int>) {
         print("executing retrieveMessages", userID, friends)
         socket.emit("initial retrieval of new messages", userID, friends)
     }
     
-    // TODO: change this to 1) encrypted message 2) noisified message --both dictionaries
-    func sendEncryptedChat(message: AnyObject){
```

Comment:
This function "sendEncryptedChat" seems like it was commented out by accident. It is needed to fetch new messages from redis. Let's uncomment this entire function!